projects/js-toolkit/packages/portal-base/src/util/promptForDeployPath.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * SPDX-FileCopyrightText: © 2021 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {FilePath, Project, format} from '@liferay/js-toolkit-core';
+import fs from 'fs';
+import {createInterface} from 'readline';
+
+const {fail, print, question} = format;
+
+export default async function promptForDeployPath(
+	project: Project
+): Promise<FilePath> {
+	const lines = createInterface({
+		input: process.stdin,
+	});
+
+	print(question`Please enter your local Liferay installation directory`);
+
+	let deployDir: FilePath;
+
+	for await (const line of lines) {
```

Comment:
Whoa, haven't seen `for await` before, very cool

Reply:
you don't know about it until you need it :sweat_smile: